functions.py

```diff
@@ -628,28 +634,29 @@ def get_to_base_unit_and_type_prefix(from_uri, query_uri):
     overlaps_sparql = """\
 PREFIX geo: <http://www.opengis.net/ont/geosparql#>
 PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX ipo: <http://purl.org/dc/terms/isPartOf> 
+PREFIX dcterms: <http://purl.org/dc/terms/>
+# PREFIX ipo: <http://purl.org/dc/terms/isPartOf> 
 PREFIX geox: <http://linked.data.gov.au/def/geox#>
-PREFIX qb4st: <http://www.w3.org/ns/qb4st/>
+# PREFIX qb4st: <http://www.w3.org/ns/qb4st/>
 PREFIX epsg: <http://www.opengis.net/def/crs/EPSG/0/>
 PREFIX dt: <http://linked.data.gov.au/def/datatype/>
 SELECT <SELECTS>
 WHERE {
     {
         { 
```

Comment:
Note this bracket might not be closed and might need commenting

Reply:
Which bracket? there is nothing in the change.